src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/NomisEntity.kt

```diff
@@ -0,0 +1,66 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa
+
+import jakarta.persistence.Column
+import jakarta.persistence.FetchType.LAZY
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.MappedSuperclass
+import org.hibernate.annotations.Generated
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.helper.EntityOpen
+import java.time.LocalDateTime
+
+@MappedSuperclass
+@EntityOpen
+class NomisEntity {
```

Comment:
subclass for any entity that wants to return audit data back to DPS

Reply:
Maybe better to have a class name about "Audit". We have loads of NOMIS entities where we're not modelling the audit columns.